pom.xml

```diff
@@ -746,8 +727,12 @@
 					<artifactId>maven-javadoc-plugin</artifactId>
 					<version>3.3.1</version>
 					<configuration>
+						<encoding>utf8</encoding>
 						<source>11</source>
-						<additionalparam>${javadoc.opts}</additionalparam>
+						<doclint>none</doclint>
```

Comment:
I tried this setup locally, leaving out only this line, and that reproduced the "unknown tag" error we are also seeing on Jenkins. Adding this in silences it :man_shrugging: